pkg/cmd/pipelinerun/describe.go

```diff
@@ -168,7 +168,16 @@ func hasFailed(pr *v1alpha1.PipelineRun) string {
 	}
 
 	if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
-		return pr.Status.Conditions[0].Message
+		for _, taskrunStatus := range pr.Status.TaskRuns {
```

Comment:
:thinking: I wonder if this needs a fix upstream (so that the `pr.Status.Conditions[0].Message` is clearer) *or* if we should display "both" (aka: having the pr message if not empty, and having the first taskrun message between `()`) ?

Reply:
Yes this looks better, will do the necessary changes :+1:

Reply:
I will update to `prMessage (trMessage)`

Reply:
@vdemeester PR updated.